sites/main-footer/inc-report-problem-en.html

```diff
@@ -1,50 +1,55 @@
-		<form action="#">
-			<fieldset>
-				<legend><span class="field-name">Please select all that apply:</span></legend>
-				<div class="checkbox">
-					<label for="gc-rprt-prblm-f1">
-						<input type="checkbox" name="problem1" value="Yes" id="gc-rprt-prblm-f1" />A link, button or video is not working
-					</label>
-					<input name="problem1" type="hidden" value="No" />
-				</div>
-				<div class="checkbox">
-					<label for="gc-rprt-prblm-f2">
-						<input type="checkbox" name="problem2" value="Yes" id="gc-rprt-prblm-f2" />It has a spelling mistake
-					</label>
-					<input name="problem2" type="hidden" value="No" />
-				</div>
-				<div class="checkbox">
-					<label for="gc-rprt-prblm-f3">
-						<input type="checkbox" name="problem3" value="Yes" id="gc-rprt-prblm-f3" />Information is missing
-					</label>
-					<input name="problem3" type="hidden" value="No" />
-				</div>
-				<div class="checkbox">
-					<label for="gc-rprt-prblm-f4">
-						<input type="checkbox" name="problem4" value="Yes" id="gc-rprt-prblm-f4" />Information is outdated or wrong
-					</label>
-					<input name="problem4" type="hidden" value="No" />
-				</div>
-				<div class="checkbox">
-					<label for="gc-rprt-prblm-f5">
-						<input type="checkbox" name="problem5" value="Yes" id="gc-rprt-prblm-f5" />I can't find what I'm looking for
-					</label>
-					<input name="problem5" type="hidden" value="No" />
-				</div>
-				<div class="checkbox">
-					<label for="gc-rprt-prblm-f6">
-						<input type="checkbox" name="problem6" value="Yes" id="gc-rprt-prblm-f6" />Other issue not in this list
-					</label>
-					<input name="problem6" type="hidden" value="No" />
-				</div>
-			</fieldset>
-			<button type="submit" class="btn btn-primary wb-toggle" data-toggle='{"stateOff": "hide", "stateOn": "show", "selector": ".gc-rprt-prblm-tggl"}'>Submit</button>
-		</form>
+<details class="brdr-0">
+	<summary class="btn btn-default text-center">Report a problem on this page</summary>
+	<div class="well row">
+		<div class="gc-rprt-prblm">
+			<div class="gc-rprt-prblm-frm gc-rprt-prblm-tggl">
+				<form action="#">
+					<fieldset>
+						<legend><span class="field-name">Please select all that apply:</span></legend>
+						<div class="checkbox">
+							<label for="gc-rprt-prblm-f1">
+								<input type="checkbox" name="problem1" value="Yes" id="gc-rprt-prblm-f1" />A link, button or video is not working
+							</label>
+							<input name="problem1" type="hidden" value="No" />
+						</div>
+						<div class="checkbox">
+							<label for="gc-rprt-prblm-f2">
+								<input type="checkbox" name="problem2" value="Yes" id="gc-rprt-prblm-f2" />It has a spelling mistake
+							</label>
+							<input name="problem2" type="hidden" value="No" />
+						</div>
+						<div class="checkbox">
+							<label for="gc-rprt-prblm-f3">
+								<input type="checkbox" name="problem3" value="Yes" id="gc-rprt-prblm-f3" />Information is missing
```

Comment:
I suggest to match our current report of problem web form that was published on Canada.ca. I know it got updated a few time since but let's document the switch from this version directly to the latest one.

Reply:
@duboisp Just making sense of this. Are you asking to create a documentation page for Report a problem? This request is pushed by a deadline, I think we should document it later.

Reply:
Yes a very basic one that will contains at least a snapshot of the HTML markup associated with a version number.
I know the version 3 is down the pipe and let to fully document it when we will start the formal GCWeb integration dev work.

Reply:
done!